index.html

```diff
@@ -58,6 +59,216 @@ <h1 class="identity_text">I'm Simba, I'm glad to see you!</h1>
+          <p class="technology_text">html</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+
+    <div class="project">
+      <img src="Snapshoot_Portfolio2.png" class="project_img" alt="project_image">
+      <h2 class="project_title">Tonic</h2>
+      <ul class="project_details">
+        <li class="list_items">
+          <p class="list_items_text">CANOPY</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">Back End Dev</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">2015</p>
+        </li>
+      </ul>
+      <p class="project_description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="technology_used">
+        <li class="technology">
+          <p class="technology_text">html</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+
+    <div class="project">
+      <img src="Snapshoot_Portfolio3.png" class="project_img" alt="project_image">
+      <h2 class="project_title">Multi-Post Stories</h2>
+      <ul class="project_details">
+        <li class="list_items">
+          <p class="list_items_text">CANOPY</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">Back End Dev</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">2015</p>
+        </li>
+      </ul>
+      <p class="project_description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="technology_used">
+        <li class="technology">
+          <p class="technology_text">html</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
```

Comment:
## OPTIONAL

- [ ] Nice job so far 👍, But still missing something. Please follow the Figma design

- [ ] Original design: **Here, you can see both sides (marked in red lines) of the design at once. the design is stable**. 
![zz](https://user-images.githubusercontent.com/63972670/159112530-6c8890bc-c282-43f7-b792-3546cb3685fc.JPG)

- [ ] Your Design: **From here, you cannot see both sides(marked in red lines) at once, unless you use the horizontal scroll to see both sides.** 

![zz1](https://user-images.githubusercontent.com/63972670/159112909-9da55308-9c2c-4cc6-ba8c-0b7d32c0dd8e.jpeg)

![zz2](https://user-images.githubusercontent.com/63972670/159112917-f2ee91d5-8b53-420d-80b4-3e32c88c3092.jpeg)